scripts/signing_secure_boot_prod.sh

```diff
@@ -0,0 +1,2 @@
+# In this script Vendor should code  the logic to build a secure boot image by using production vendor flows.
```

Comment:
Do we still need this file now?? The Vendor scripts will be anyway outside the git repo.

Reply:
fixed

Reply:
Ack